build/npm/navigation-react-native/navigation.react.native.d.ts

```diff
@@ -211,6 +211,10 @@ export interface SearchBarProps {
      * The search field text
      */
     text?: string;
+    /**
+     * The search field background color
+     */
+    barTintColor?: string;
```

Comment:
Can you copy this change into [navigation-react-native.d.ts file in the test folder](https://github.com/grahammendick/navigation/blob/master/NavigationReactNative/test/node_modules/%40types/navigation-react-native.d.ts)